### DISCUSSION TITLE
Determinations Tab - added functionality

### DISCUSSION THREAD

**Discussion by STHerbariumLSU** [Symbiota/community-board#512] [Wish List]
Can there be more functionality added to the Determinations Tab?  A few things: 
1. make edits searchable through "date modified".  This is so helpful for checking work.
2. display "family" in the dets. page
3. sync the occ tab and dets tab better so that updates occur when editing either page
4. in the dets tab, don't make determiner or date determined required.  It's not required in the occ. tab.
5. can the Taxon Remarks field not roll over with the determinations?  It makes sense that ID References and ID Remarks rolls over with the Sci Name, but the Taxon field is used more broadly, at least at our institution.  We use it for common names, filed as, checklist identifiers, and general comments.  If others use it more specifically to the sci name, it should probably roll over.  And, it should be added to the Dets tab.
6. I've had trouble with dets not rolling over in the past, but this issue hasn't popped up lately.  Just to keep in mind if work is done here.

Thanks!

**Comment by ljwalker** [comment]
A user from TORCH additionally requests the ability to search on both accepted an unaccepted determinations added to this tab.

**Comment by fbungartz** [comment]
I agree with most but definitely NOT that tacon remarks don't roll over. Taxon remarks must NOT be used to enter common names (Zhere us a fiield for this on the taxin editor and tacon profile editor.

Taxon Remarks are meant for taxonomic comments concerning the identification. Fir example why an identification is problematic (qualifier: cf.), why a specimen has been identified only to a group or might otherwise be problematic... etc.
